tests/server/login-test.js

```diff
@@ -0,0 +1,29 @@
+const app = require('../../tools/app.js')
+const request = require('supertest')
+const assert = require('assert')
+
+describe('login', () => {
+
+  it('should attempt to login the test/asdf user', done => {
+    request(app)
```

Comment:
@Bronson-Brown-deVost : I like supertest because you write nice, fluent tests like this with integrated assertions.